azkaban-common/src/main/java/azkaban/server/JettyServerUtils.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package azkaban.server;
+
+import azkaban.utils.Props;
+import java.util.List;
+import org.apache.log4j.Logger;
+import org.mortbay.jetty.bio.SocketConnector;
+import org.mortbay.jetty.security.SslSocketConnector;
+
+public class JettyServerUtils {
+
+  private static final int DEFAULT_HEADER_BUFFER_SIZE = 10 * 1024 * 1024;
+  private static final Logger logger = Logger.getLogger(JettyServerUtils.class);
```

Comment:
Will we be able to use slf4j logger here?

Reply:
Done. Utilized org.slf4j.LoggerFactory#getLogger(java.lang.Class<?>)